code/modules/shareddream/dream_entry.dm

```diff
@@ -1,9 +1,10 @@
 var/list/dream_entries = list()
 
 /mob/living/carbon/human
+	var/shared_dream = FALSE
```

Comment:
Why not just. Use `bg` here? `bg` is set to `null` whenever `shared_dream` is set to `FALSE`; and set to non-`null` whenever `shared_dream` is set to `TRUE`.